conventional-changelog-conventionalcommits appears to be missing constants.js file

The package makes use of the constants file here

https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-conventionalcommits/index.js#L3

but doesn't appear to include it for distribution here

https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-conventionalcommits/package.json#L15

which results in this error when using it in v7

```
Error: Cannot find module './constants'
Require stack:
- /home/tim/.nvm/versions/node/v18.17.1/lib/node_modules/conventional-changelog-conventionalcommits/index.js
- /home/tim/Documents/coding/teedup.ai/document-service/noop.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:121:18)
    at Object.<anonymous> (/home/tim/.nvm/versions/node/v18.17.1/lib/node_modules/conventional-changelog-conventionalcommits/index.js:3:34)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Module.require (node:internal/modules/cjs/loader:1143:19) {
  code: 'MODULE_NOT_FOUND',
```

Is this expected?

--- a/packages/conventional-changelog-conventionalcommits/package.json
+++ b/packages/conventional-changelog-conventionalcommits/package.json
@@ -19,7 +19,8 @@
     "index.js",
     "parserOpts.js",
     "writerOpts.js",
-    "templates"
+    "templates",
+    "constants.js"
   ],
   "author": "Ben Coe",
   "engines": {

